apps/ocas/src/openc2_handler.erl

```diff
@@ -135,28 +128,29 @@ has_action(true, Req, State ) ->
     JsonMap = maps:get(json_map, State2),
     ActionBin = maps:get( <<"action">>, JsonMap ),
     lager:info("action bintext: ~p", [ActionBin] ),
-    { ActionValid, ActionValue } = actions:is_valid_action(ActionBin),
+    { ActionValid, {ActionModule, ActionFunction} } = actions:is_valid_action(ActionBin),
```

Comment:
Line 131 is too long:     { ActionValid, {ActionModule, ActionFunction} } = actions:is_valid_action(ActionBin),.